mono/mcs/class/System.Windows.Forms/System.Windows.Forms.CocoaInternal/XplatUICocoa.cs

```diff
@@ -1138,7 +1138,7 @@ internal override IntPtr CreateWindow (CreateParams cp)
 		
 		internal override void DoEvents() {
             MSG msg = new MSG ();
-			while (PeekMessage (null, ref msg, IntPtr.Zero, 0, 0, (uint)PeekMessageFlags.PM_REMOVE)) {
+			while (PeekMessage (null, ref msg, IntPtr.Zero, 0, int.MaxValue, (uint)PeekMessageFlags.PM_REMOVE)) {

```

Comment:
PeekMessage is documented to pump all messages when both `wFilterMin` and `wFilterMax` are zero, so there's no need to change it.